core/src/App.html

```diff
@@ -769,7 +792,7 @@
   let alerts = [];
 
   const getAlertWithId = (alertQueue, id) => {
-    if (!alertQueue || !alertQueue.length) return;
+    if (!alertQueue || !alertQueue.length > 0) return;
```

Comment:
Wrong check... in this context it always leads to the correct result, but be aware that 
!x > 0 is not the same as !(x > 0) for negative x!

Reply:
Good point! Thanks for reviewing it.